packages/client/test/net/protocol/snapprotocol.spec.ts

```diff
@@ -82,7 +85,56 @@ tape('[SnapProtocol]', (t) => {
     t.end()
   })
 
-  t.test('verify that GetStorageRanges handler encodes/decodes correctly', (t) => {
+  t.test('AccountRange should encode/decode correctly',(t)=>{
+    const config = new Config({ transports: [] })
+    const chain = new Chain({ config })
+    const p = new SnapProtocol({ config, chain })
+    /* eslint-disable @typescript-eslint/no-use-before-define */
+    const data=RLP.decode(Buffer.from(accountRangeRLP,'hex')) as unknown
+    const {
+          reqId,
+          accounts,
+          proof
+        } = p.decode(p.messages.filter((message) => message.name === 'AccountRange')[0], data);
+    const payload = RLP.encode(p.encode(p.messages.filter((message) => message.name === 'AccountRange')[0], {reqId,accounts,proof}));
+    t.ok(accountRangeRLP===Buffer.from(payload).toString("hex"),"Re-encoded payload should match with original")
+    t.end()
+
+  })
+
+  t.test('GetAccountRange/AccountRange should verify real sample',async(t)=>{
+    const config = new Config({ transports: [] })
+    const chain = new Chain({ config })
+    const p = new SnapProtocol({ config, chain })
+    /* eslint-disable @typescript-eslint/no-use-before-define */
+    const reqData= RLP.decode(Buffer.from(getAccountRangeRLP,'hex'));
```

Comment:
@faustbrian why are these semicolon additions not caught by linting? Did this get lost somewhere along the way? Any insight?

Reply:
@faustbrian yea i push with `--no-verify` but after running lint:fix, which is sort of my primary workflow to push. And the `lint:fix` doesn't seem to be prettifying/formating these things. would really appreciate if we can fix  the "lint:fix"